### ISSUE TITLE
.pmdl not working?

### ISSUE THREAD

**Issue by eouia** [eouia/MMM-Assistant#5]
I have tried thousand times with `.pmdl` but failed. `.umdl` works well. I don't know why.

**Comment by cowboysdude** [comment]
"Most likely, you have a mis-matched microphone, or you are using your pmdl for other people. It's also possible that you have noise or click sound around your recorded audios, and removing those manually should improve the performance quite a bit."

**Comment by E3V3A** [comment]
@cowboysdude  Can you elaborate? I don't understand your explanation.
